### DISCUSSION TITLE
don't offer image uploading in Rich Text notes until the feature is implemented

### DISCUSSION THREAD

**Discussion by chaserene** [standardnotes/forum#2668] [Suggestions]
**Describe the bug**
the "Picture" functionality in Rich Text notes offers uploading and embedding from the local machine, only to follow with the error message that this is not presently supported. I had a user complain to me about this. displaying the upload selector obviously creates an expectation to be able to do this, so I suggest hiding it until the feature is implemented. (also, the link to the [Help entry](https://standardnotes.com/help/36/how-do-i-attach-encrypted-files-to-my-notes) in the error message is not clickable, but that's irrelevant if you get rid of the upload selector altogether.)

**To Reproduce**
see above

**Expected behavior**
see above

**Desktop (please complete the following information):**
 - OS: GNU/Linux AppImage
 - Version: 3.23.241

**Comment by effieeee** [comment]
Hi @chaserene, indeed we've yet to implement a way to display file and image attachments inline 😞 Apologies for the confusion!
